best_buy_ruby.gemspec

```
@@ -27,16 +27,16 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   # Production dependencies
-  spec.add_dependency 'activesupport', '~> 6.0.2.1'
-  spec.add_dependency 'faraday', '~> 1.0.0'
+  spec.add_dependency 'activesupport', '~> 6.0'
```

This will only work with rails 6.x

What do you think if we add support for more versions?
```suggestion
  spec.add_dependency 'activesupport', ['>= 4', '< 7']
```
Those are the versions we are supporting in yaaf.

---

Absolutely!